JujuWindowsUtils/JujuWindowsUtils.psm1

```diff
@@ -352,18 +352,38 @@ function Install-WindowsFeatures {
     )
     PROCESS {
         $rebootNeeded = $false
-        foreach ($feature in $Features) {
-            $state = Start-ExecuteWithRetry -Command {
-                Install-WindowsFeature -Name $feature -ErrorAction Stop
-            }
-            if ($state.Success -eq $true) {
-                if ($state.RestartNeeded -eq 'Yes') {
-                    $rebootNeeded = $true
+            foreach ($feature in $Features) {
+                if (Get-IsNanoServer) {
```

Comment:
Can you add a .SYNOPSIS documenting this function? Something like:

This function installs windows features. For Nano, you already need to have the features installed, and this function merely enables them.